aea/configurations/base.py

```diff
@@ -812,22 +861,26 @@ def package_dependencies(self) -> Set[ComponentId]:
     @property
     def json(self) -> Dict:
         """Return the JSON representation."""
-        return {
-            "name": self.name,
-            "author": self.author,
-            "version": self.version,
-            "license": self.license,
-            "aea_version": self.aea_version,
-            "fingerprint": self.fingerprint,
-            "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
-            "class_name": self.class_name,
-            "protocols": sorted(map(str, self.protocols)),
-            "restricted_to_protocols": sorted(map(str, self.restricted_to_protocols)),
-            "excluded_protocols": sorted(map(str, self.excluded_protocols)),
-            "dependencies": self.dependencies,
-            "description": self.description,
-            "config": self.config,
-        }
+        return OrderedDict(
+            {
+                "name": self.name,
+                "author": self.author,
+                "version": self.version,
+                "description": self.description,
+                "license": self.license,
+                "aea_version": self.aea_version,
+                "fingerprint": self.fingerprint,
+                "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
+                "class_name": self.class_name,
+                "protocols": sorted(map(str, self.protocols)),
```

Comment:
In the spirit of the above, I would move protocols above class_name, I would also have class_name, 
 config, excluded_protocols, restricted_to_protocols in this order. I'd have dependencies last.